lib/blueflood_http_client.js

```diff
@@ -145,8 +148,19 @@ function buildPayloads(tenantId, timestamp, gauges, counters, timers, sets, flus
       // if objLength > realMax, we will end up sending a bundle that is too large. we'll let the client deal with
```

Comment:
This comment needs to be updated or removed to align with modification below...

**however**, re-reading this led me to discover the client code does already log the payload of the failed request here
https://github.com/rackerlabs/blueflood-statsd-backend/blob/master/lib/blueflood_http_client.js#L80

Along those lines, since this plugin as a whole can't know how BF's Repose has configured the request payload limit, it might be presuming too much and be knowingly lossy to not make the attempt at all.